src/test/kotlin/org/rust/lang/core/completion/RsCompletionTest.kt

```diff
@@ -1199,4 +1199,42 @@ class RsCompletionTest : RsCompletionTestBase() {
     """, """
         fn foo(f: &FnOnce(/*caret*/)) {}
     """)
+
+    fun `test complete true in variable decl`() = doSingleCompletion("""
```

Comment:
I'd personally prefer to extract these tests into separate class
Just to make test search more convenient and don't create too large files

Reply:
Extracted